rust/auction/program/src/processor.rs

```diff
@@ -102,6 +102,8 @@ pub struct AuctionDataExtended {
     // Unimplemented fields
     /// Tick size
     pub tick_size: Option<u64>,
+    /// instant sale price
+    pub instant_sale_price: Option<u64>,
```

Comment:
When adding a field to an existing struct you need to put it at the end, so it is backwards compatible with other structs like it. Place this after gap tick size percentage.